pkg/apis/camel/v1/trait/registry.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 
 package trait
 
+// WARNING: The Tracing trait is **deprecated** and will removed in future release versions.
```

Comment:
I guess you meant `Registry` instead of `Tracing`

Reply:
Yeah, a regeneration is undergoing. In any case, we have a nightly process taking care to fix any generation mistake.